test/data/repositories/rhel-84.json

```diff
@@ -0,0 +1,15 @@
+{
+    "module_platform_id": "platform:el8",
+    "distribution": {
+      "name": "rhel-84",
+      "description": "Red Hat Enterprise Linux (RHEL) 8"
+    },
+    "x86_64": {
+      "image_types": [ "ami", "vhd", "rhel-edge-commit", "rhel-edge-installer" ],
+      "repositories": [{
+        "baseurl": "http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.5/compose/BaseOS/x86_64/os/"
```

Comment:
The repo does not pin to rhel 8.4 internal repo, but rhel 8.5. Your image is not RHEL 8.4, but RHEL 8.5.
@yih-redhat You have to re-run the test after you pin image-builder repo to rhel 8.4.

Reply:
Hmm, this is the repo I got from Achilleas...
I will change to 8.4, and see if it works.

Reply:
changed to 8.4, and passed. 
but another test job failed, which has nothing to do with my change...
will try to rerun.

Reply:
rerun this pr, passed.